src/docs/development/platform-integration/apple-watch.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Adding an Apple Watch extension"
+description: "Learn how to add an Apple Watch target to a Flutter app"
+---
+
+While you cannot build an Apple Watch app with Flutter, it is possible to add a native Apple Watch extension to a Flutter app.
+
+## Step 1: Enable bitcode in Xcode
+Apple Watch targets require bitcode to be enabled, so follow the steps [here](https://github.com/flutter/flutter/wiki/Creating-an-iOS-Bitcode-enabled-app-(experimental)) to use bitcode in your app.
```

Comment:
Hi, @eospi! Thanks for your PR! 

Linking on "here" is bad form. Can you rephrase to:
Apple Watch targets require bitcode to be enabled, so follow the steps in [Creating an iOS Bitcode enabled app][]
to use bitcode in your app.

[Creating an iOS Bitcode enabled app]: LINK (The link in your text doesn't work for me. Can you double check?)